build/mbl/build-update-payloads.py

```diff
@@ -90,11 +90,11 @@ def main():
         bitbake.run_command(command, check=True, verbose=True)
 
     # Create the payloads for bootloader components, kernel and rootfs
-    bootloader1_base_path = args.outputdir / "bootloader1_payload"
-    bootloader2_base_path = args.outputdir / "bootloader2_payload"
-    kernel_base_path = args.outputdir / "kernel_payload"
-    rootfs_base_path = args.outputdir / "rootfs_payload"
-    multi_component_base_path = args.outputdir / "multi_component_payload"
+    bootloader1_base_path = args.outputdir / "bootloader1"
+    bootloader2_base_path = args.outputdir / "bootloader2"
+    kernel_base_path = args.outputdir / "kernel"
+    rootfs_base_path = args.outputdir / "rootfs"
+    multi_component_base_path = args.outputdir / "multi-component"
```

Comment:
Can we put a (working) app into the multi-component payload please?

@rwalton-arm, will the multi-component update test be able to handle that?

Reply:
> Can we put a (working) app into the multi-component payload please?
> 
> @rwalton-arm, will the multi-component update test be able to handle that?

After https://github.com/ARMmbed/mbl-core/pull/249 is merged it will be able to handle it.

Reply:
Addressed here: c1f0addf2c9d0bde00da0e3152cd7552779ebc8b

Reply:
Thanks